install/rpm/pga/config.inc.php

```diff
@@ -85,6 +85,15 @@
 // passworded local connections.
 $conf["extra_login_security"] = true;
 
+// If extra session security is true, then PHP's session cookies will have
```

Comment:
Same for here

Reply:
moving to pgafix branch, please see sep. PR